CHANGELOG.md

```diff
@@ -0,0 +1,178 @@
+
+# Changelog
+
+- [24.5.1 - 2024-10-31](#2451---2024-10-31)
+- [24.5.0 - 2024-08-16](#2450---2024-08-16)
+
+All notable changes to this project are documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
+
+## 24.5.1 - 2024-10-31
+
+- ### Added
+  
+  - **Data Abstraction**
+    - New Bitwise Math Operations Added `PinSiteData<T>`
+      - `ShiftLeft(int count)`
+        - Return Type: `PinSiteData<T>`
+        - Description: Shifts the value to the left by the specified bit count, for each element, per site. The count must be positive. This method is supported only when the underlying data type of the `PinSiteData<T>` object, `T`, is an integer type.
```

Comment:
Want to mention the operator like you do for bitwise AND (&)?
"Shifts the value to the left **(<<)** by the specified bit count"

Same comment to ShiftRight and to SiteData shift methods.